### PR TITLE
issue#3801 - replaced the old StatisticBox with the Statistic for the blood DK module

### PR DESCRIPTION
**Pull request by leozhekov**
Part 1 from the full replacement. I decided to separate it by spec modules for easier PR review.

### CODE REVIEW

**Comment by Khazakdk** [comment]
Can you include a screenshot of the new rune tracker statistic?  The diff on the snapshots makes it look like there's been visual changes to it and its helpful to have a screenshot on PRs with visual changes

File: src/parser/deathknight/blood/modules/features/BloodPlagueUptime.js [hunk 1]
```diff
@@ -39,15 +41,18 @@ class BloodPlagueUptime extends Analyzer {
...
-        value={`${formatPercentage(this.uptime)} %`}
-        label="Blood Plague uptime"
-      />
+      <Statistic
+        size="small"
+        position={STATISTIC_ORDER.CORE(2)}
+      >
+        <BoringSpellValueText spell={SPELLS.BLOOD_PLAGUE}>
+          <>
+            <UptimeIcon /> {formatPercentage(this.uptime)}% <small>Blood Plague uptime</small>
```
Comment (joshinat0r) [inline src/parser/deathknight/blood/modules/features/BloodPlagueUptime.js]: Would prefer if this was just `uptime`

File: src/parser/deathknight/blood/modules/features/BoneShield.js [hunk 2]
```diff
@@ -51,35 +49,41 @@ class BoneShield extends Analyzer {
...
+                  </tr>
+                ))}
+              </tbody>
+            </table>
+          </>
+        )}
+      >
+        <BoringSpellValueText spell={SPELLS.BONE_SHIELD}>
+          <>
+            <UptimeIcon /> {formatPercentage(this.uptime)}% <small>Bone Shield uptime</small>
```
Comment (joshinat0r) [inline src/parser/deathknight/blood/modules/features/BoneShield.js]: Would prefer just `uptime` aswell

File: src/parser/deathknight/blood/modules/features/CrimsonScourge.js [hunk 3]
```diff
@@ -95,15 +97,19 @@ class CrimsonScourge extends Analyzer {
...
-        label="Crimson Scourge procs wasted"
+      <Statistic
+        position={STATISTIC_ORDER.CORE(6)}
+        size="flexible"
         tooltip={`${this.wastedCrimsonScourgeProcs} out of ${this.crimsonScourgeProcsCounter} procs wasted.`}
-      />
+      >
+        <BoringSpellValueText spell={SPELLS.CRIMSON_SCOURGE}>
+          <>
+            {formatPercentage(this.wastedCrimsonScourgeProcsPercent)} % <small>Crimson Scourge procs wasted</small>
```
Comment (joshinat0r) [inline src/parser/deathknight/blood/modules/features/CrimsonScourge.js]: Would prefer just `procs wasted`

File: src/parser/deathknight/blood/modules/features/MarrowrendUsage.js [hunk 4]
```diff
@@ -196,10 +195,15 @@ class MarrowrendUsage extends Analyzer {
             Avoid casting Marrowrend unless you have {this.refreshAtStacks} or less stacks or if Bone Shield has less than 6sec of its duration left.
           </>
         )}
-      />
+      >
+        <BoringSpellValueText spell={SPELLS.MARROWREND}>
+          <>
+            {this.badMRCasts} / {this.totalMRCasts} <small>Bad Marrowrend casts</small>
```
Comment (joshinat0r) [inline src/parser/deathknight/blood/modules/features/MarrowrendUsage.js]: Same, `bad casts`

**Review by joshinat0r — CHANGES_REQUESTED** [review]
Just a minor UI update, the icon is a bit bigger. 
Old:
image

New:
image


Might be nitpicky, but the only "complaint" I have would be the length of the descriptions. It used to look like this where the icon of the statisticbox would be displayed along with the label.
image
Now the box has the icon and spell name already in the box, repeating "*Crimson Scourge* procs wasted" seems a bit redundant, especially if its killing the layout on some resolutions.
image

Other than that, looks good and should be got to go after those minor updates.
Thanks for contributing!

File: src/parser/deathknight/blood/modules/features/MarrowrendUsage.js (hunk 4 above)
Comment (leozhekov) [reply src/parser/deathknight/blood/modules/features/MarrowrendUsage.js]: I agree with your comments and I'll make the changes later today.

**Pull request merged**